src/scss/main.scss

```diff
@@ -4,3 +4,176 @@
 .create-schedule {
   display: none;
 }
+
+.nav{
+  margin:0px;
+}
+
+.headline{
+  background-color:#80D8FF;
+  color:white;
+  width:300px;
+  height:70px;
+  text-align:center;
+  padding-top:20px;
+  font-size:20px;
+}
+
+body, html{
```

Comment:
Just for code style; I like to keep my really general selectors (like `body` and `html` at the very top of the file, with more specific selectors like `.headline` below.